tweb/src/main/frontend/src/js/classes/Utils.ts

```diff
@@ -33,6 +33,23 @@ export default class Utils {
         return window.location.hash ? window.location.hash.split('#')[1] : '';
     }
 
+    public static setQueryParam(key, value) {
+        let url: URL = new URL(location.href);
+        let query: URLSearchParams = url.searchParams;
+        if (!value) {
```

Comment:
I like the shorthand notation: 
value ? query.set(key, value) : query.delete(key);

Reply:
Yep, I prefer that too, but seems I still manage to forget it sometimes..